images/ci-search/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang
+WORKDIR /go/src/github.com/openshift/ci-search
+COPY . .
```

Comment:
Not sure what are you copying here?! I guess search tool code, can you got clone in here instead of cp

Reply:
Yes, it is the search tool code. As I was using their Dockerfile as a reference, I forgot to change this.
Will modify and submit.